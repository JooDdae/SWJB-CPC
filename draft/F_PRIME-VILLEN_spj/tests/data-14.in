1699201762871493109160112903442129373
