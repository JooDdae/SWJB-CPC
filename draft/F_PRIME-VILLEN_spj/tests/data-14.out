15
266 306 818 35 2 29 0 110 0 420 601 10 2 4 2 